core/src/main/java/org/apache/brooklyn/location/winrm/WinRmMachineLocation.java

```diff
@@ -18,32 +18,6 @@
  */
 package org.apache.brooklyn.location.winrm;
 
-import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
I prefer not having changes that just re-order imports.

Also, the vast majority of our code uses the Eclipse convention for ordering imports (this looks like the Intellij default). I believe that (some) Intellij users change their settings to use the Eclipse ordering, for Brooklyn.